Changed: Updated Traefik's confuration documentation (#39)
discord-tickets/docs: docs/self-hosting/reverse-proxy.md

Reviewer: 1. `(...)` should be commented out
2. some of the options (like ports and volumes) aren't needed
3. Use inline comments on the labels, like the compose file example at https://github.com/discord-tickets/docs/blob/main/docs/self-hosting/installation/docker.md

Refer to the documentation linked above for more information.

<div class="annotate" markdown>

```yaml title="docker-compose.yml" hl_lines="12 14-18 22"
version: "3.9"

services:
  mysql:
    #(...)
  bot:
    #(...)
    networks:
      - discord-tickets
      - {==traefik_network==} # (1)!
    #(...)
      HTTP_TRUST_PROXY: "true" # (2)!
    labels:
      - "traefik.enable=true" # (3)!
      - "traefik.docker.network={==traefik_network==}" # (4)!
      - "traefik.http.routers.tickets.entrypoints{==websecure==}" # (5)!
      - "traefik.http.routers.tickets.rule=Host(`{==tickets.example.com==}`)" # (6)!
      - "traefik.http.services.tickets.loadbalancer.server.port=8169" # (7)!

networks:
  discord-tickets:
  {==traefik_network==}: # (1)!
    external: true
#(...)

```

</div>

1. Replace the traefik_network by the network used by traefik to reverse_proxy & loadbalancing your services
2. Set to true if you're using a reverse proxy
3. Enables Traefik for this container

> 4. Optionnal but reccomanded, tells traefik the docker network to use

Reviewer: ```suggestion
4. Optional but recommended, tells Traefik which Docker network to use
```

### Configuration

> This configuration assure you're traefik configuration is correctly done and that you know it.

Reviewer: ```suggestion
This example shows the additions you may need to make to your `docker-compose.yml` file to configure Traefik.
```

> Change highlighted values to correspond to your traefik configuration, refering to the traefik's documentation

Reviewer: ```suggestion
After installing and configuring Traefik (referring to the documentation linked above), change the highlighted values to match your configuration.
```
